### PR TITLE
Fixes #31383 - Add report template for CVEs

### PR DESCRIPTION
**Pull request by xprazak2**
A new report template that shows vulnerabilities
detected during an OVAL scan.

Needs https://github.com/theforeman/foreman_openscap/pull/458

### CODE REVIEW

File: app/views/unattended/report_templates/host_-_vulnerabilities.erb
Comment (xprazak2) [inline app/views/unattended/report_templates/host_-_vulnerabilities.erb:22]: It would probably be good to constrain this by version.

**Comment by ares** [comment]
works nicely, let's wait for the foreman_openscap release with the patches, but ready to merge afterwards

**Comment by ares** [comment]
still works fine, note for me: before merge double check the macro name hasn't changed

File: app/views/unattended/report_templates/host_-_vulnerabilities.erb
Comment (ares) [inline app/views/unattended/report_templates/host_-_vulnerabilities.erb]: please avoid empty lines, it would break the final CSV (or other format)

File: app/views/unattended/report_templates/host_-_vulnerabilities.erb
Comment (ares) [inline app/views/unattended/report_templates/host_-_vulnerabilities.erb]: here to, no new line please

File: app/views/unattended/report_templates/host_-_vulnerabilities.erb
Comment (ares) [inline app/views/unattended/report_templates/host_-_vulnerabilities.erb]: one of the stories asked for hosts filter, could you please also add it here? just define a new input of search type and use it in `load_hosts` macro as `search: input('Hosts filter')`

**Review by ares — CHANGES_REQUESTED** [review]
few details

**Comment by xprazak2** [comment]
Updated, no longer blocked.

**Review by ares — APPROVED** [review]
Thanks @xprazak2, merging!

**Comment by ezr-ondrej** [comment]
@xprazak2 would you mind demoing it? :)

**Pull request merged**
